Gems/OpenXRVk/Code/Include/OpenXRVk/InputDeviceXRController.h

```diff
@@ -413,6 +437,13 @@ namespace AzFramework
         Implementation* GetImplementation() const;
 
     protected:
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        // AzFramework::DebugDisplayEventBus interface
+#if !defined(_RELEASE)
+        void CheckDebugDrawCheat();
```

Comment:
Nice. Add a quick comment explaining the purpose of the methods.

Reply:
Sure thing.